### DISCUSSION TITLE
Quotes not being escaped in social metadata

### DISCUSSION THREAD

**Discussion by glenlong** [arthelokyo/astrowind#179] [General]
I have a blog post title with quotes in it. Here's the frontmatter:

```
---
title: "5 Podia Community \"Hacks\" I Just Can't Live Without"
description: Podia's community feature might be simple but it's certainly not basic. And these "hacks" take it to the next level.
publishDate: June 12, 2023
image: ~/assets/images/blog/podia-community-hacks.png
category: podia
---
```

I believe I am escaping the quotes correctly as far as markdown is concerned, but in the final HTML the quotes aren't escaped:

`<meta property="og:title" content="5 Podia Community "Hacks" I Just Can't Live Without" />`
 
 The result is that the title is truncated when shared on social media.

Any ideas?

Glen

**Comment by prototypa** [comment]
Hello @glenlong 

You are right! There was a problem with the @astrolib/seo dependency, I updated it to the latest version and now it escapes the characters. Thank you so much.

**Reply by glenlong** [reply]
Great  - thank you!
